libshvbroker/src/rpc/clientconnectiononbroker.cpp

```diff
@@ -276,6 +282,31 @@ void ClientConnectionOnBroker::onRpcDataReceived(shv::chainpack::Rpc::ProtocolTy
 		}
 		if(m_idleWatchDogTimer)
 			m_idleWatchDogTimer->start();
+		if (isSlaveBrokerConnection() && cp::RpcMessage::isResponse(md) && m_slaveLsRequestId.count()) {
+			int request_id = cp::RpcMessage::requestId(md).toInt();
+			auto it = std::find_if(m_slaveLsRequestId.begin(), m_slaveLsRequestId.end(), [&request_id](const RequestId &r) {
+				return r.requestId == request_id;
+			});
+			if (it != m_slaveLsRequestId.end()) {
+				QVector<int> caller_ids = makeCallerIdList(cp::RpcMessage::callerIds(md));
+				if (caller_ids == it->callerIds) {
+					m_slaveLsRequestId.erase(it);
+					cp::RpcValue resp = decodeData(protocol_type, msg_data, 0);
+					if (!resp.at(cp::RpcMessage::MetaType::Key::Error).isValid()) {
```

Comment:
what `if (resp.at(cp::RpcMessage::MetaType::Key::Error).isValid()) ` ?

Reply:
Nothing, reponse is not modified and is proceeded vid default handler.